handler/src/test/java/io/netty/handler/ssl/SniHandlerTest.java

```diff
@@ -320,7 +320,7 @@ public void testFallbackToDefaultContext() throws Exception {
                 ch.writeInbound(Unpooled.wrappedBuffer(message));
                 // TODO(scott): This should fail because the engine should reject zero length records during handshake.
                 // See https://github.com/netty/netty/issues/6348.
-                // fail();
+                fail();
```

Comment:
@njhill thanks... Lets see if this works for all JDK versions that we use on the CI :) If so we can close the issue as well.